miscDeps

```diff
@@ -1 +1 @@
-Subproject commit 239621041225ce15b93d6aeb96945eaf67e10898
+Subproject commit a7f918c421e91f1b1601ab822ede2c869275ef5c
```

Comment:
Cool. I won't use git add . in the future for simple things and that 
should solve most of this. Did this pr get closed because I realized I 
based it off of my master and if I push new committs there it will put 
them here.

On 3/30/2016 9:19 PM, James Teh wrote:
>
> In miscDeps 
> <https://github.com/nvaccess/nvda/pull/5842#discussion_r57996555>:
>
> > @@ -1 +1 @@
> > -Subproject commit 239621041225ce15b93d6aeb96945eaf67e10898
> > +Subproject commit a7f918c421e91f1b1601ab822ede2c869275ef5c
> You probably did git commit -a, added all files with git add or 
> something like that, but you had an outdated miscDeps submodule at the 
> time. Careful observance of git status and/or git show after the 
> commit will prevent such things.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/5842/files/ae68ffa67b8deeae01c6d60f0c8050b4ec4d05b1#r57996555>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194